<launch>
    <include file="$(find myhamster_simulation)/launch/hamster_v7.launch">
        <!--<arg name="rvizpath" value="$(find myhamster_simulation)/rviz/rviz_navigation.rviz"/>-->
        <arg name="x_pos" value="-0.5"/>
        <arg name="gazebo_world" value="$(find myhamster_simulation)/worlds/turtlebot3_stage_4.world"/>
    </include>

    <!-- move_base -->
    <include file="$(find mytutorial_pkg)/launch/move_base.launch">
        <!--<arg name="local_planner" value="dwa"/>-->
    </include>


	<!-- Map server -->
	<node name="map_server" pkg="map_server" type="map_server" args="$(find mytutorial_pkg)/map/turtlebot3_stage_4.yaml" output="screen">
        <param name="frame_id" value="map"/>
	</node>

    <!-- AMCL -->
    <include file="$(find mytutorial_pkg)/launch/amcl.launch"/>

    <!-- velocity conversion -->
    <param name="twist_cmd_topic" value="/cmd_vel"/>
    <param name="ackermann_cmd_topic" value="/ackermann_cmd"/>
    <param name="wheelbase" value="0.2"/> <!--0.165-->
    <param name="move_base/TebLocalPlannerROS/min_turning_radius" value="0.7"/><!--0.3-->
    <node pkg="mytutorial_pkg" type="cmd_vel_to_ackermann_drive_simple.py" name="velocity_convert_node"/>

    <!-- rviz -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find myhamster_simulation)/rviz/rviz_navigation.rviz"/>

</launch>